client/hosting/performance/utils.ts

```diff
@@ -6,3 +8,15 @@ export function isValidURL( url: string ) {
 		url
 	);
 }
+
+// Returns props for the support link
```

Comment:
Nitpick: The comment `// Returns props for the support link` doesn't add any value beyond what the function name itself already communicates, so I usually _try_ to avoid them.